### DISCUSSION TITLE
Practicas Intro FrontEnd

### DISCUSSION THREAD

**Discussion by romarpla** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#21] [Announcements]
Hola explorers!! @LaunchX-InnovaccionVirtual/mission-innova001 

Ya están publicadas las practicas de la primer semana de nuestra misión de FrontEnd.

Solo tienen que seguir el link

https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO/practicas

TIENEN HASTA EL 25/02/22 PARA SUBIR SUS EVIDENCIAS

***¡Vámonos hasta el espacio y más allá Explorers!***

**Comment by armap99** [comment]
Muchas gracias, solo una duda como se subieran las evidencias en un formulario al igual que las de Python o de otra forma ? @romarpla

**Reply by AngelCruzO** [reply]
Tendremos que subirlas a un repositorio de Github, al final de las 6 semanas supongo daran las instrucciones para enviarlas

**Reply by mabvmex** [reply]
Hola @AngelCruzO, en el anuncio al principio de esta discusión se dice que el dia límite es el 25 de Febrero, no al final de las 6 semanas... ¿saben bien como es el proceso de entrega?

**Reply by AngelCruzO** [reply]
@mabvmex yo puse en 6 semanas, porque si leemos y comprendemos lo que se dice la publicación, ahí menciona que se tiene hasta el 25 para "Subir" que es diferente a entregar. Y aún no se dice nada sobre eso

**Comment by AngieKings** [comment]
Una pregunta, nosotros elegimos el proyecto o se asignarán en la siguiente sesión alguna temática o restricciones?

**Reply by AngelCruzO** [reply]
Ya esta asignado el caso de estudio, es el enunciado de Abogabot

**Reply by RicardoRobledo** [reply]
> Ya esta asignado el caso de estudio, es el enunciado de Abogabot

¿Donde puedo ver eso?, ¿O en que parte lo mencionaron?

**Reply by AngelCOUACM** [reply]
Entrando al link de las prácticas en la parte de abajo

**Reply by iRetr0o** [reply]
Es el README de las prácticas

**Reply by edu18RR** [reply]
Gracias.

**Comment by AngelCruzO** [comment]
@romarpla se puede ocupar otro formato para los requerimientos o es necesario ocupar el que nos proporciono?

**Reply by MiguelGomez1997** [reply]
yo digo que si en la publicación dice "pueden"

**Reply by romarpla** [reply]
puede ser el que se les acomode

**Comment by Oliverlju** [comment]
Nice

**Comment by JacomeGerardo** [comment]
Mil disculpas, yo sé que quizás esta pregunta sea quisquillosa pero no logro comprender la manera de llenar los formatos  de las prácticas. ¿Las instrucciones las dio en algún live?Los estoy viendo pero no encuentro la manera. Gracias por su atención y comprensión
![Captura de pantalla (518)](https://user-images.githubusercontent.com/99065797/154817041-00672b6f-5712-4d1d-b820-5610b57a9b6f.png)
!

**Reply by AngelCOUACM** [reply]
No hay live sobre eso, eso lo puedes ver buscando sobre el tema de análisis de requerimientos.

**Reply by JacomeGerardo** [reply]
Gracias!!

**Reply by JacomeGerardo** [reply]
Alguna página que me recomiendes?

**Reply by AngelCruzO** [reply]
Pues no, solo que busques el tema en google, en youtube

**Reply by LeonardoAlvarezRivera** [reply]
Es algo asi https://youtu.be/evi2mhhYGws

**Reply by jdcarrillo123** [reply]
Excelente @LeonardoAlvarezRivera muy practico

**Comment by DanielOe21** [comment]
En un Live comentó que subiría un vídeo explicativo, hasta ahí me quedé, no me he enterado de algún cambio

**Reply by JacomeGerardo** [reply]
Sí, yo recordaba lo mismo por eso es que pensé eso. Pero tendremos que resolverlo por cuenta propia porque tenemos hasta el viernes 25 :(

**Comment by EMILIO425** [comment]
Ahmmm, ese link para que servirá?, No entendí exactamente

**Reply by AngelCOUACM** [reply]
El la no de Github es para ver las especificaciones del caso de estudio, así como las prácticas que se tienen que realizar. Ya que las termines, cada quien sabra si realiza su propio repositorio, clona o forkea el repo original.

**Comment by joseEnrique9889** [comment]
Hola buenos días una pregunta. En dónde encuentro los vídeos explicativo para la realización de la práctica. Si no mal recuerdo en live nos dijo que subiría unos vídeos explicativo para guiarnos y poder realizarlo.

**Reply by ghost** [reply]
tambien me interesa saber

**Reply by AngelCruzO** [reply]
Aun no se suben, les sugiero ir investigando por su cuenta. En los en vivos se mencionaron algunos ejemplos para poder hacer las practicas, excepto lo de Analisis de rquisitos

**Comment by Alexandre-Hernandez** [comment]
Buenas noches, me podrías explicar el proceso de entrega y el día límite por favor, es que yo veo las regrebaciones en los fines de semana, porque entre semana estudio.

Get Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by villegas97** [reply]
La fecha limite para subir las practicas es el 25, pero no han dicho donde.

**Comment by Alexandre-Hernandez** [comment]
Muchas gracias.

Get Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by RafBD** [comment]
No entiendo que hay que hacer 😣 Alguien me podría explicar por favor.

**Reply by BaniPlfx** [reply]
X2, Ayuda

**Reply by eduardocanoperez** [reply]
lee la intro: https://github.com/romarpla/FrontEnd-Mision/tree/main/01%20-%20INTRO 
de ahí ve la grabación del 16 de febrero, la encuentras en teams, 

la actividad es practicar lo que nos explican en el video.

lo que entregaremos ===> https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO/practicas

**Comment by charlyguz** [comment]
sigo si entender que se tiene que hacer exactamente, si alguien sabe por favor explíquenme, espero que en el live explique que se tiene que hacer por que las instrucciones son muy vagas

**Reply by Fullgamer494** [reply]
Pues hablando de una manera muy superficial, tenemos que resolver las prácticas del repositorio que nos mandó, y por lo que he visto todo es con base a la documentación dada en la INTRO del repositorio FrontEnd-Mission, no hay explicación detallada, ni instrucciones especificas dadas en los lives, toca resolverlo por iniciativa, al menos eso he entendido

**Reply by roge64** [reply]
La idea es ponerte en los zapatos de un FrontEnd developer que participa en un proyecto solicitado por una firma de abogados. El proyecto busca automatizar las demandas de sus clientes, en otras palabras, que cada cliente genere de forma automática la demanda en el sistema o aplicación que tú vas a desarrollarle a la firma de abogados. 
En esta etapa es más un trabajo de analista de sistemas para lograr definir los requerimientos del cliente. El formato que nos proporcionan es una guía para documentar este proceso y que las partes involucradas definan el alcance del proyecto.  

Es un poco confuso, sobre todo  si no hemos tenido experiencia participando en algún tipo de proyecto, pero básicamente en esta etapa vamos a conocer las expectativas del cliente y a definir lo que la empresa de desarrollo (nosotros) se compromete a desarrollar y a entregar. 

Imagina que quieres hacer tu casa y el responsable de la obra (Arquitecto) se sienta contigo y te pregunta si quieres la casa de dos plantas o de una, cuantas recámaras, baños, tipo de pisos, acabados, etc.. Igual te pregunta la cantidad de dinero de la que dispones, entre otras cosas. 

Al final te va a decir si lo que buscas es factible tomando en cuenta todos los factores y obvio el costo.


Espero que te sea de ayuda y si tienes dudas con gusto te apoyo.

Saludos!

**Reply by mgc14** [reply]
Entonces todavía no se hace nada oficial en HTML para entregar,¿verdad?

**Reply by JacomeGerardo** [reply]
> Entonces todavía no se hace nada oficial en HTML para entregar, ¿verdad?

No, aún no

**Comment by Alexandre-Hernandez** [comment]
La neta yo tampoco, pero en live de hoy hay que preguntar dudas.

Get Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by roge64** [reply]
La idea es ponerte en los zapatos de un FrontEnd developer que participa en un proyecto solicitado por una firma de abogados. El proyecto busca automatizar las demandas de sus clientes, en otras palabras, que cada cliente genere de forma automática la demanda en el sistema o aplicación que tú vas a desarrollarle a la firma de abogados.
En esta etapa es más un trabajo de analista de sistemas para lograr definir los requerimientos del cliente. El formato que nos proporcionan es una guía para documentar este proceso y que las partes involucradas definan el alcance del proyecto.

Es un poco confuso, sobre todo si no hemos tenido experiencia participando en algún tipo de proyecto, pero básicamente en esta etapa vamos a conocer las expectativas del cliente y a definir lo que la empresa de desarrollo (nosotros) se compromete a desarrollar y a entregar.

Imagina que quieres hacer tu casa y el responsable de la obra (Arquitecto) se sienta contigo y te pregunta si quieres la casa de dos plantas o de una, cuantas recámaras, baños, tipo de pisos, acabados, etc.. Igual te pregunta la cantidad de dinero de la que dispones, entre otras cosas.

Al final te va a decir si lo que buscas es factible tomando en cuenta todos los factores y obvio el costo.

Espero que te sea de ayuda y si tienes dudas con gusto te apoyo.

Saludos!

**Comment by iaeloko** [comment]
Es como orientado a las necesidades del cliente y no tratar de resolver el código en esta etapa, y más adémate hacer el código con lo que sigamos aprendiendo ?

Yael Villarreal

**Comment by iaeloko** [comment]
Es sin hacer código aun? Solo plasmar cómo lo haríamos de acuerdo a las necesidades del cliente?

Yael Villarreal

**Reply by EdgarMolina23** [reply]
Osease ahorita estamos haciendo lo que un analista de sistemas haria ya el dia de hoy se empieza con el html que ahora si ya seria la parte de frontend

**Reply by AngelCruzO** [reply]
Ojo esto de la parte de análisis y diseño, también es frontend. HTML y css es maquetación de frontend y Js es darle funcionamiento a la maquetación

**Reply by DiegoPenalozaJuarez** [reply]
obviamente es sin codigo pues no hemos visto absolutamente nada de codigo aun, esto es muy importante la toma de requerimientos, ya en una empresa formal esto es el pan de cada dia de los proyectos y es lo que diferencia de un verdadero desarrollador a solo un programador que desgraciadamente en el mercado la mayoria solo son programadores les falta mucho ver el tema del negocio.

**Reply by JuanSalvador73** [reply]
De hecho, en la mayoría de las empresas es así, cuando construyes un sitio rara vez como desarrollador haces - o ves siquiera - los requerimientos para un sitio web. Incluso en empresas grandes pasan por alto el proceso. Pero es buenos que nos lo enseñen.

**Comment by LisandroHdz** [comment]
En esta pagina pueden crear su Buyer Persona: 
https://www.hubspot.com/make-my-persona

**Reply by erickmatz9101** [reply]
Te lo agradezco mucho, fue muy útil y se entiende más como poder hacerlo. muchas gracias

**Comment by erickmatz9101** [comment]
Hola compañeros, ahora que estamos por iniciar con la parte del código quisiera hacer una publicación para ustedes con algunas extensiones principalmente de Visual Studio Code que me han funcionado para desarrollar. Me gustaría saber si les interesa que les haga una publicación más detallada con ellas y para que sirven ya que considero que tanto para el front como para el back serán muy útiles. Saludos a todos y todas.

**Reply by Ingrzs** [reply]
Adelante hazlo :D

**Comment by retroskateboard** [comment]
Si! Mándala :)

**Comment by Jesus-H-Gonzalez** [comment]
Hola explorers!!! espero que mi comentario les sirva para avanzar con las katas de esta misión. Este es mi punto de vista como mercadólogo ya que muchos de los términos que se usan aquí igualmente se usan en estudios de mercado. 

1. En cuanto a los **requerimientos del cliente** lo que piden es escribir en algún documento (no necesariamente el que compartió el mission comander) lo que "Abogabot", el cliente al que se le va a diseñar la página web, necesita en su página web y entre más específico mejor, es como si tuvieras una pastelería y el cliente te pide un pastel de chocolate, de dos pisos, con la leyenda "felicidades" en el segundo piso y con decoración de plantas comestibles. Eso es lo que se debe documentar pero enfocado a una página web.
2. En  **buyer persona** puede ser un poco confuso si no se tienen conocimientos de mercadotecnia pero básicamente lo que se debe hacer es describir a tu "cliente perfecto" una descripción física y mental de ese cliente, por ejemplo: quieren abrir un gimnasio para mujeres que se dediquen al fisicoculturismo y ahora imaginen a su cliente perfecto parado en la puerta de ese gimnasio; es una mujer que viste ropa deportiva, lleva una maleta grande donde guarda ropa y algunos suplementos alimenticios, es una mujer musculosa que también procura mucho su piel, por lo general lleva comida saludable en tupers y el tiempo mínimo de su estancia es de dos horas.
3. El **público objetivo** yo lo entiendo más como una segmentación de mercado, que básicamente es describir a grandes rasgos el perfil de las personas para las que se va a diseñar, en este caso, la página web; es algo parecido al "buyer persona" pero más específico y personal en el sentido que no se debe hacer como tal una descripción física de la persona, más bien una descripción demográfica(sexo, edad, ingresos mensuales), geográfica (en que país, estado, ciudad y hasta colonia vive el cliente), psicográfica( sus preocupaciones, aspiraciones en la vida, algunas costumbres que tenga, entre muchas más), y conductuales ( ¿Cómo reacciona a algunos escenarios?, que busca en un producto, ¿conoce el producto?) para que quede más claro les dejo este link --->https://www.oberlo.com/media/1603891662-segmentacion-de-mercado.png?fm=webp&w=1824&fit=max
o buscar más sobre segmentación de mercados para que quede más claro.
4. Y en cuanto a Ux y Ui no lo entiendo al 100% así que para no confundir a nadie prefiero no hacer ningún comentario.

Al final esto es un curso de Web development y supongo que sólo les interesa que nos familiaricemos con los términos y sepamos para qué se usan, y básicamente todo esto funciona cómo una guía en todo el desarrollo y tener un objetivo claro.

**Reply by BaniPlfx** [reply]
Muchas gracias!!

**Reply by JuanSalvador73** [reply]
Excelente aporte, mil gracias.

**Reply by Vickmagar** [reply]
Gracias por tu aportacion tan valiosa

**Comment by contjoseph** [comment]
Buenas noches, 
   Aun no tengo acceso a la sección de HTML. Me falta hacer algo? Gracias de antemano.

**Reply by iRetr0o** [reply]
Aún no está activado

**Reply by JuanSalvador73** [reply]
Aún no está activado, y es la 1 de la mañana del día 23 al escribir esto. Saludos.

**Comment by miguelruizF** [comment]
La extensión que yo les recomiendo para vscode es la de live server, les
crea un servidor para que puedan visualizar su trabajo sin necesidad de ir
al archivo guardado en algún lado de su equipo.

**Comment by jucegor** [comment]
Hola, comprendo que la entrega es la creación del repositorio en donde estarán las evidencias, pero no comprendo si debemos pasar la liga del repositorio en algun lugar, alguien me podria ayudar?

**Reply by DiegoHC06** [reply]
Si, de hecho se realizara un formulario como se hizo con las katas, y dentro de ello te pide la liga URL de tu repositorio. solo esperar que publiquen el formulario,

**Reply by iononi** [reply]
Hola que tal, así es debemos crear un repositorio donde subiremos nuestras evidencias. En cuanto a donde pondremos el link del repositorio eso será en un Microsoft Forms como se hizo en la primera misión (Onboarding). Sin embargo, todavía no está disponible este formulario, así que solo hay que estar pendientes del teams pues ahí el MC publicará el formulario. De momento solo debemos ir realizando las prácticas. Saludos!

**Reply by jucegor** [reply]
Oh ya excelente, gracias por la aclaración

**Comment by RealMeEdr** [comment]
Una vez hecho tu repositorio y ahí puestos tus katas.


En el teams hay un formulario que debes llenar y en ese formulario debes
poner el link de tu repositorio.
De igual manera se te específica en la publicación de teams

**Reply by mgc14** [reply]
¿En que parte esta el formulario? No se ve.

**Reply by franciscoxcode** [reply]
Yo creo que está hablando del formulario para las katas anteriores, las de Fer. El formulario para la misión nueva todavía no está disponible aún, según lo que yo entiendo.

**Reply by mgc14** [reply]
Gracias Francisco, pensé lo mismo, solo quería corroborarlo.

**Comment by davidjuarezr** [comment]
Si, por favor gracias

**Comment by roge64** [comment]
Buenas noches.

Creo que alguien ya compartió esto, pero lo incluyo por si algunos no lo han visto.

[](https://www.hubspot.es/make-my-persona)

Nos enseña lo que es un "buyer persona" pero lo mejor es que cuenta con un generador de  "buyer persona" que nos  lleva paso a paso a diseñar la nuestra.

Espero lo encuentren de utilidad.

**Reply by roge64** [reply]
Olvide la liga: https://www.hubspot.es/make-my-persona

**Reply by mgc14** [reply]
Oye, una pregunta.

¿Ese buyer persona lo tenemos que hacer con base en el ejercicio que hizo de los abogados o se tiene que hacer a nuestra elección? Eso no me quedó claro.

**Reply by roge64** [reply]
Si tú elegiste el ejemplo de los abogados, lo tienes que hacer  para ese ejemplo.

Si elegiste otro  tipo de app con clientes diferentes, entonces lo haces para lo que elegiste.

Yo estoy usando el ejemplo que publicó, y si te sirve, lo puedes ver en mi repositorio. Ya lo tengo publicado., sencillo pero creo que contiene lo esencial.

**Comment by Alex-Canela** [comment]


**Reply by Burstniart** [reply]
Porque comentan en este post que es un anuncio para la organizacion

**Comment by GuillermoESIME06** [comment]
X2

Get Outlook para Android<https://aka.ms/AAb9ysg>

________________________________
De: Alejandro Canela ***@***.***>
Enviado: miércoles, 23 de febrero de 2022 9:32 p. m.
Para: LaunchX-InnovaccionVirtual/FrontEnd-Mision
Cc: GuillermoESIME06; Team mention
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Practicas Intro FrontEnd (Discussion #21)


Ah!!!! Por que siguen llegando correos si ya le di cancelar a las discusiones, enserió que es molesto estar recibiendo tanto correo. Lo voy a poner como spam.



Enviado desde mi Samsung Mobile de Telcel



-------- Mensaje original --------
De: Rogelio Valdés ***@***.***>
Fecha: 23/2/2022 9:06 p. m. (GMT-06:00)
A: LaunchX-InnovaccionVirtual/FrontEnd-Mision ***@***.***>
CC: Alejandro Canela ***@***.***>, Manual ***@***.***>
Asunto: Re: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Practicas Intro FrontEnd (Discussion #21)


Olvide la liga: https://www.hubspot.es/make-my-persona

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/21#discussioncomment-2240912>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ATK2PSEQFB6566R4C45TAJLU4WODBANCNFSM5OZ3CEKA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

—
Reply to this email directly, view it on GitHub<https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/21#discussioncomment-2240976>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXW2HUAE2KL56VDQNOKDYCLU4WREXANCNFSM5OZ3CEKA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by AngelECC** [comment]
Por favor, para eso está la sección de discussion en git. Cualquier duda la pueden postear ahí sin problema, este medio es para que el MC nos comparta información relevante. Gracias.

Get Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by OntiverosLuis** [comment]
Por mas que leo y vuelvo a leer, veo una y otra vez los lives y sigo sin entender el publico objetivo y la toma de requerimientos...Que es lo que exactamente debo hacer???? De antemano muchas gracias MC. Enviado desde Correo para Windows De: RomarplaEnviado: viernes, 18 de febrero de 2022 10:24 p. m.Para: LaunchX-InnovaccionVirtual/FrontEnd-MisionCC: OntiverosLuis; Team mentionAsunto: [LaunchX-InnovaccionVirtual/FrontEnd-Mision] Practicas Intro FrontEnd (Discussion #21) Hola explorers!! @LaunchX-InnovaccionVirtual/mission-innova001Ya están publicadas las practicas de la primer semana de nuestra misión de FrontEnd.Solo tienen que seguir el linkhttps://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/tree/main/01%20-%20INTRO/practicasTIENEN HASTA EL 25/02/22 PARA SUBIR SUS EVIDENCIAS¡Vámonos hasta el espacio y más allá Explorers!—Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you are on a team that was mentioned.Message ID: ***@***.***>

**Comment by MiguelAngelAguilar312** [comment]
Buenos días cual es link para poder subir la información o nuestra practicas de la semana 1.

**Reply by YYosefRidana** [reply]
x2, creo que aun no se publica ni nada :C

**Reply by romarpla** [reply]
[Link de publicación de Teams](https://teams.microsoft.com/l/message/19:8a61b3259c3147259ba0197e9b103432@thread.tacv2/1645814720628?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1645814720628&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%201%20-%20Despegue%20%F0%9F%9A%80&createdTime=1645814720628)

Este es el link dentro del equipo de Teams para que puedan ver el link del Formulario

**Reply by CRISTIANDENA91** [reply]
¿Cuáles son las prácticas? Estoy confundido

**Comment by ParisCarmona** [comment]
Mission Commander, excelente dia!

disculpame la confusion, es correcto no hay algo de orden en
mis repositorios.



te compartiré mi situación y sin modo de queja o drama.

tengo actualmente varias tareas que me demandan atención
en el 2009 funde mi negocio despues de haber trabajado en un proyecto en el
banco, en este tiempo le proporcionaba soluciones de arrendamiento de
equipo de computo para outsourcing, para finales del 2019 ya no tuve
contratos por la nueva ley del trabajo y los equipos que se compraron a
credito se nos quedaron por no poder arrendar a outsourcing, despues de la
pandemia nos comieron algunos acuerdos y personal tuvo que salirse de
trabajar, retome un negocio de lavaderia y me quede sin personal en la
lavanderia, actualmente soy la unica persona que se hace cargo del negocio
de la lavanderia, compre el año pasado unos equipos para poner una
hamburgueseria y no es posible por ahora que se heche andar ese negocio por
lo mismo de la pandemia, todo esta explicacion es par compartirte que
atiendo varias mis actividades de vender servicios de soluciones en soporte
hardware y almismo tiempo , atiendo mi negocio de la lavanderia, yo me
inscribi y a mediados del mes de febrero pude comenzar a leer y conocer
algo de la plataforma, apenas estoy terminando de conocer la plataforma
poniendome al corriente con algo de las katas de Sherpad Fernanda Ochoa,
estoy terminando de ver algunos ejemplos y retomando viejas enseñanzas en
la escuela, pero el tiempo destinado a esto si es poco, permiteme poder
ponerme al corriente y poder compartirte el resultado de tu guia, espero
esta situacion no me afecte en mi intecion de participar y no perder esta
oportunidad.

gracias por tu tiempo Mission Commander

**Comment by R44wk** [comment]
Hola Explorers.
Algun host gratuito que me recomienden aparte de "https://www.000webhost.com/"
Gracias

**Reply by lordfriky** [reply]
Dependiendo tus necesidades GitHub tiene un host gratuito muy básico llamado "GitHub Pages"!

**Reply by AngelCruzO** [reply]
000webhost te lo recomiendo solo si ya vas a dejar un solo sitio, porque no te deja crear mas que un sitio para la parte gratuita. Al menos para el Frontedn Github Pages es la mejor opción

**Reply by R44wk** [reply]
Les agradezco, probare GithubPages porque en 000webhost estoy baneado y si es bueno

**Reply by pakosanchez27** [reply]
Espero y te sirva amigo yo eso este [Netlify](https://app.netlify.com) no es el mejor pero esta muy bueno 😁

**Reply by richirrim** [reply]
Usa tu propio repo para crear un host para tu page.

1. Te ubicas en el repo de tu proyecto. 
2. Clic en "Setting".
3. En la parte izquierda hay un item que dice "Page".
4.  Donde dice 'Source' haces clic en 'main' y por ultimo selecciones la que dice '/(root)'.

Y así ya tendrás un enlace para abrir tu page en un host gratuito de Github pages. Ojo, no seas impaciente sino aparece, tarda un poco en que la página se puede visualizar correctamente.

**Reply by R44wk** [reply]
Grax @richirrim, eso me funciono

**Reply by modestcap** [reply]
Saludos, veo que nadie lo ha mencionado, pero microsoft ofrece uno gratuito de hasta 10 sitios web teniendo una cuenta gratuita de Azure, puedes encontrarlo como Azure static pages. Lo puedes encontrar en el siguiente sitio web https://portal.azure.com/

**Reply by AngelCruzO** [reply]
@FernandoHernandezR2049 Yo lo pensaba mencionar, pero es parte del temario. Mejor dejar que llegue ese tema y lo explique mejor el MC

**Comment by Mavoportentous** [comment]
Puedes ponerlo en Github y desplegarlo en Vercel. Así le hice, como
comentan con Github pages también funciona.

**Reply by R44wk** [reply]
Si, probare en github pages, grax

**Comment by R44wk** [comment]
Te agradezco francisco

**Comment by abarbizan** [comment]
Reinicie mi computadora y ya no puedo entrar a teams ni en app ni en web
alguien sabe qué pasó o cómo puedo solucionarlo

**Reply by iRetr0o** [reply]
Por qué no te deja entrar a Teams?

**Reply by BattiestWinter** [reply]
A mi me pasó algo parecido hace una semana. Busca que exista una actualización disponible de tu equipo en  configuración --> actualización y seguridad, con eso queda jaja. 

Ya no usen esta discusión para dudas por fa :((

**Comment by DiegoAndresRa** [comment]
Si
